### PR TITLE
Update ShenyuExtConfiguration.java

### PR DESCRIPTION
**Pull request by key9527**
此处注入的 new RemoteAddressResolver() 会使得 org/apache/shenyu/springboot/starter/gateway/ShenyuConfiguration.java:162的ForwardedRemoteAddressResolver注入失效。 
这与官方文档的说明不符，官网文档显示在Apache ShenYu网关中获取IP的自带实现为：org.apache.shenyu.web.forward.ForwardedRemoteAddressResolver。

Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor-guide).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `./mvnw clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-web/src/main/java/org/apache/shenyu/web/configuration/ShenyuExtConfiguration.java [hunk 1]
```diff
@@ -50,7 +51,6 @@ public ShenyuResult<?> shenyuResult() {
     @Bean
     @ConditionalOnMissingBean(value = RemoteAddressResolver.class, search = SearchStrategy.ALL)
     public RemoteAddressResolver remoteAddressResolver() {
-        return new RemoteAddressResolver() {
-        };
+        return new ForwardedRemoteAddressResolver(1);
```
Comment (yu199195) [inline shenyu-web/src/main/java/org/apache/shenyu/web/configuration/ShenyuExtConfiguration.java:54]: this code alreay in ShenyuConfiguration.java. 

can remove this ?

Comment (neo-shirley) [reply shenyu-web/src/main/java/org/apache/shenyu/web/configuration/ShenyuExtConfiguration.java:54]: yes，you can delete this code。

Comment (yu199195) [reply shenyu-web/src/main/java/org/apache/shenyu/web/configuration/ShenyuExtConfiguration.java:54]: you can remove this (modify this pr)

**Review by neo-shirley — COMMENTED** [review]
@yu199195  dear author，as you suggest, I have deleted this conflict code

File: shenyu-web/src/main/java/org/apache/shenyu/web/configuration/ShenyuExtConfiguration.java [hunk 2]
```diff
@@ -20,6 +20,7 @@
 import org.apache.shenyu.plugin.api.RemoteAddressResolver;
 import org.apache.shenyu.plugin.api.result.DefaultShenyuResult;
 import org.apache.shenyu.plugin.api.result.ShenyuResult;
+import org.apache.shenyu.web.forward.ForwardedRemoteAddressResolver;
```
[the remark this answers is not in the document]
Comment (key9527) [reply shenyu-web/src/main/java/org/apache/shenyu/web/configuration/ShenyuExtConfiguration.java]: sorry，my fault,  i forget this import code，i have deleted this，please merge again。

**Comment by yu199195** [comment]
hi， pls check ci，

> ShenyuExtConfigurationTest.testRemoteAddressResolver:58->lambda$testRemoteAddressResolver$1:59 » NoSuchBeanDefinition

**Comment by key9527** [comment]
> hi， pls check ci，
> 
> > ShenyuExtConfigurationTest.testRemoteAddressResolver:58->lambda$testRemoteAddressResolver$1:59 » NoSuchBeanDefinition

ok,  All checks have passed

**Review by yu199195 — APPROVED** [review]

**Pull request merged**